### DISCUSSION TITLE
How can I create a multi language documentation with GitBook?

### DISCUSSION THREAD

**Discussion by techfreak01** [GitbookIO/community#703] [1. How to] [answered]
I want to use GitBook to create a product documentation in multiple languages.
The end user should be able to select one of the available languages to see the documentation in that language.
I saw that GitBook AI can be used to translate pages into other languages and I want to use that.

Is that possible? Where can I find documentation on how to do it?

**Comment by addisonschultz** [comment]
Hey @techfreak01 - you'll want to start here to learn more about publishing multiple versions of a docs site! https://docs.gitbook.com/published-documentation/publish-multiple-spaces-on-one-site

Regarding the translations - you can use GitBook AI - but keep in mind it's only part of our Pro plan: https://www.gitbook.com/pricing

At this time, it will translate selections of text/pages - translating an entire site at once is not possible yet

**Comment by techfreak01** [comment]
Can I select the text of a whole page then with GitBook AI and save the result as a page in the space of the other language?You should better document that translation feature, since it cannot be tried without subscribing for the pro plan, but may motivate to subscribe for the pro plan.It would be a great feature for the future to translate a complete space into multiple other languages and even update the translated spaces later after making changes in the main language space.Regards, MatthiasAm 26.08.2024 um 03:24 schrieb Addison ***@***.***>:﻿
Hey @techfreak01 - you'll want to start here to learn more about publishing multiple versions of a docs site! https://docs.gitbook.com/published-documentation/publish-multiple-spaces-on-one-site
Regarding the translations - you can use GitBook AI - but keep in mind it's only part of our Pro plan: https://www.gitbook.com/pricing
At this time, it will translate selections of text/pages - translating an entire site at once is not possible yet

**Reply by addisonschultz** [reply, marked as answer]
We have it documented here! https://docs.gitbook.com/content-editor/editing-content/write-and-edit-with-ai

And yes - you can copy and paste the result of translating an entire page to a new space!
